### PR TITLE
EPMEDU-627: Add More screen content

### PR DESCRIPTION
**Pull request by igorTimofiejczyk**

### CODE REVIEW

File: animeal/src/Flows/Main/MorePartition/Model/MorePartitionModel.swift [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        switch mode {
+        case .donate:
+            return PartitionContentModel(
+                header: PartitionContentModel.Header(title: "Donate"),
+                content: PartitionContentModel.Content(actions: nil),
+                footer: PartitionContentModel.Footer(
+                    action: PartitionContentModel.Action(
+                        actionId: PartitionContentModel.Action.ActionID.copyIBAN,
+                        title: "Copy IBAN",
+                        type: PartitionContentModel.FooterType.acccent,
```
Comment (VictorSemenchuk) [inline animeal/src/Flows/Main/MorePartition/Model/MorePartitionModel.swift]: Typo aCCCent

File: animeal/src/Flows/Main/MorePartition/Model/MorePartitionModel.swift [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    // MARK: - Initialization
+    init() { }
+
+    // MARK: MorePartitionModelProtocol
+    func fetchContentModel(_ mode: PartitionMode) -> PartitionContentModel {
+        // TODO: Localize strings!!!
+        switch mode {
+        case .donate:
+            return PartitionContentModel(
+                header: PartitionContentModel.Header(title: "Donate"),
```
Comment (VictorSemenchuk) [inline animeal/src/Flows/Main/MorePartition/Model/MorePartitionModel.swift:15]: Do we need localization?

**Review by VictorSemenchuk — APPROVED** [review]

**Comment by VictorSemenchuk** [comment]
clap-applause

**Review by diantyn — APPROVED** [review]

File: animeal/src/Flows/Main/MorePartition/Model/MorePartitionModel.swift (hunk 1 above)
Comment (igorTimofiejczyk) [reply animeal/src/Flows/Main/MorePartition/Model/MorePartitionModel.swift]: Thanks nice catch!

File: animeal/src/Flows/Main/MorePartition/Model/MorePartitionModel.swift (hunk 2 above)
Comment (igorTimofiejczyk) [reply animeal/src/Flows/Main/MorePartition/Model/MorePartitionModel.swift:15]: yep, in next PR

**Review by VictorSemenchuk — APPROVED** [review]

**Pull request merged**